.forge-snapshots/swap with hooks.snap

```diff
@@ -1 +1 @@
-104143
\ No newline at end of file
+103987
```

Comment:
it just makes no sense 😭

Reply:
wonder if running `forge test --isolate` before and after this change will still introduce a diff

Reply:
good q i'll try it

Reply:
nope sadly they all still change - different values ofc but they still all change by like <200 ish. Could be that thats an expected result but its definitely weird